fahadmak/stackoverflow_clone
# #167138197 Create and List Comments for a answers

**What does this PR do?**
This PR ensures that a user can create a comment and list comments on an answer.

**Description of Task to be completed?**
- create CreateAnswerCommentForm Form
- create AnswerComment model
 - create comment answer form in question_detail.html template

**What are the relevant Pivotal Tracker stories?**
[#167138197](https://www.pivotaltracker.com/n/projects/2364284/stories/167138197)

**How should be tested manually?**
Run each of the following commands in your terminal in the respective order.

- `git clone https://github.com/fahadmak/stackoverflow_clone.git`
- `cd stackoverflow_clone`
- `git checkout ft-create-list-question-comments-167138181`
- `virtualenv venv`
- `. venv/bin/activate`
- `pip install -r requirements.txt`
- `run migrations using this command python manage.py migrate`
- `run python manage.py runserver 127.0.0.1:8000`
- `Go to this link in your browser 127.0.0.1:8000`
- `You will be redirected to the login page where you should log in`
- `Click on the 'Register Today' link to go to sign up page`
- `Try to sign up an account and then use it in the login page`
- `Click on a question to go to the question_detail page`
- `Click on comment button in the answer card to open comment form`
- `Write a comment in the form`

**Screenshots**
<img width="1021" alt="Screen Shot 2019-07-17 at 8 51 55 AM" src="https://user-images.githubusercontent.com/13882936/61350429-2a3daf80-a870-11e9-8746-61a32779db8f.png">
